publisher/gcp/publisher_test.go

```diff
@@ -62,3 +65,26 @@ func TestEventBusAsync(t *testing.T) {
 
 	testutil.EventPublisherCommonTests(t, publisher1, publisher2)
 }
+
+// setUpEventPublisher is a helper that creates a new GCP publisher for @appID
+func setUpEventPublisher(appID string) (*EventPublisher, error) {
+	var projectID = "looplab-eventhorizon"
+
+	// See https://developers.google.com/identity/protocols/application-default-credentials
+	if creds := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS"); creds != "" {
+		var m = make(map[string]string)
```

Comment:
Use the := form and allocate with {} instead of `make`. Also move to the row above where it's first used.